remove-node.yml

```diff
@@ -5,7 +5,7 @@
     ansible_ssh_pipelining: true
   gather_facts: true
 
-- hosts: etcd:k8s-cluster:vault:calico-rr
+- hosts: "{{ node }}"
```

Comment:
I am not opposed here. Let's default it though. Please change the line to:
```
hosts: "{{ node | default('etcd:k8s-cluster:vault:calico-rr') }}"
```

Reply:
i can change this, but may i ask why? do you want to delete all nodes? or want to use `--limit` without per/post tasks?

Reply:
because you should not break previous behavior, there're other users.

Reply:
ok, change will come.